### ISSUE TITLE
Sandy Drumming Bear

### ISSUE THREAD

**Issue by MKorin** [WarEmu/WarBugs#11857]
Dont give 5% initiative

![1](https://user-images.githubusercontent.com/13823154/41195188-5cd77c84-6c29-11e8-81a6-1c249cad1e03.jpg)
![2](https://user-images.githubusercontent.com/13823154/41195189-5d9179fe-6c29-11e8-83c5-d6810a71609d.jpg)
![3](https://user-images.githubusercontent.com/13823154/41195192-5eae105e-6c29-11e8-91c5-3135555c5b4b.jpg)

**Comment by thorbolt80** [comment]
I remember lheana told me that the team dont want to  implement this buff yet with all the pets. Need confirm....

**Comment by Ryanfawkez** [comment]
That's correct; vanity pets won't get any sort of buff associated to them. Their tooltips have to be adjusted and updated still.
